aten/src/ATen/core/interned_strings.h

```diff
@@ -96,6 +96,7 @@ namespace c10 {
   _(aten, _set_item)               \
   _(aten, index_put_)              \
   _(aten, device)                  \
+  _(aten, hash)                    \
```

Comment:
We haven't always been following this, but I think it would be a nice rule of thumb that reserved keywords go under the prim namespace